lib/fastlane/plugin/android_emulator/actions/android_emulator_action.rb

```diff
@@ -16,33 +16,51 @@ def self.run(params)
         system("#{adb} emu kill > /dev/null 2>&1 &")
```

Comment:
I think we should just make the closing of the emulator blocking.
The sleep was more a workaround.

Therefore we should not need the additional `sleep` or `error_callback`

Reply:
I made the emulator kill blocking, but the `error_callback` was still called! I think there's a timing issue with the calls to wait for the emulator to start and the boot animation to finish. For some reason, they return with a 1 exit status instead of 0 when the emulator is slow to close.

Still I've removed a couple of sleep commands which is good :-)